app/templates/app/cc-question.html

```diff
@@ -31,7 +34,29 @@ <h4>Marked for draft</h4>
 </div>
 {# IDE goes here #}
 {% if student %}
-<div id="IDE">IDE goes here!</div>
+
+<div id="IDE">
+<p>Select Theme</p>
+<select onchange="selectTheme()" id="theme-select">
+  <option selected="">default</option>
+  <option>abcdef</option>
+  <option>base16-light</option>
+  <option>blackboard</option>
+  <option>cobalt</option>
+  <option>colorforth</option>
+  <option>monokai</option>
+</select>
+
+<textarea id="editor"></textarea>
+<button id="run" onclick="judge_code(
+'{{ question.id }}', '{{ question.assignment.language }}')">Run</button>
+<button id="submit" disabled onclick="submit_code(
+{{ question.id }}, {{ student.id }}, {{ assignment.id }})">Submit</button>
+</div>
```

Comment:
Seems hacky. Use [`json_script`](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#json-script) template tag filter and then get the required values from JS.

Reply:
This is raising "Object Not Serizable Error". I wiil try to find a fix

Reply:
Do one thing, create a `dict` and store the things required for code execution in it. Then you can refer it in the template.